src/parameter_entry_class.F90

```diff
@@ -488,7 +493,7 @@ subroutine get_vector_character (this, value, errc)
     type is (character(*))
 #ifdef INTEL_DPD200255963
       if (allocated(value)) deallocate(value)
-      allocate(value(size(v)), source=v)
+      allocate(character(len(v(1))) :: value(size(v)))
```

Comment:
Just len(v), and you are missing the assignment "value = v".
Instead of modifying the Intel workaround to also work with GNU, let's leave the Intel alone and just add
#elifdef GNU_54070  (is there such a thing?)
     if (allocated(value)) ...
     allocate(character(len(v)) :: ...)
     value = v